### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by NathanTP**
1.9.0 release to coordinate with upcoming Chipyard release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,39 @@
 This changelog follows the format defined here: https://keepachangelog.com/en/1.0.0/
 Versioning follows semantic versioning as described here: https://semver.org/spec/v2.0.0.html
 
+## [1.9.0] - 2021-05-21
+This is largely a maintenance release with a few minor features and a bunch of
+bug fixes. The most significant change is a bump to Linux 5.7rc3. The new
+'firesim-dir' configuration option is also signficant because it enables more
+flexible deployment of FireMarshal in chipyard and FireSim and is required for
```
Comment (abejgonzalez) [inline CHANGELOG.md:10]:
```suggestion
flexible deployment of FireMarshal in Chipyard and FireSim and is required for
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,39 @@
...
+* PR #129 adds a number of new features to help support profiling and debug. In
+  particular, Marshal now saves the raw kernel ELF file including debug symbols
+  (BBL strips DWARF info). It also handles a few handy scripts for running
+  FirePerf tools.
+* PR #141 adds the 'post-bin' script option to workloads. This is primarily
+  useful for building custom drivers but may have other uses.
+
+### Changed
+* PR #139 enables RVC in the kernel by default
+* PR #143 Changes the behavior of kernel fragments. Previously, workload kfrags
```
Comment (abejgonzalez) [inline CHANGELOG.md:26]:
```suggestion
* PR #143 changes the behavior of kernel fragments. Previously, workload kfrags
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,39 @@
...
+  were applied directly on top of the distro's default config. Now kfrags are
+  inherited through the chain of parents.
+* PR #151 bumps us to Linux 5.7rc3
+* PR #148 switches to the SBI serial driver from the sifive uart due to a
+  baud-rate configuration issue. This will be reversed eventually.
+
+### Fixed
+* PR #126 adds detection of changes in overlay directories that were missed before
+* PR #128 Fixes an issue when calling FireMarshal inside a makefile
```
Comment (abejgonzalez) [inline CHANGELOG.md:35]:
```suggestion
* PR #128 fixes an issue when calling FireMarshal inside a makefile
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,39 @@
...
+* PR #133 fixes the --workdir option that had broken (and adds tests so it won't happen again)
+* PR #135 Jobs were not including device drivers
```
Comment (abejgonzalez) [inline CHANGELOG.md:37]:
```suggestion
* PR #135 jobs were not including device drivers
```

**Comment by alonamid** [comment]
Are you trying to get #152 in before the release? (other than that, LGTM as well)

**Comment by abejgonzalez** [comment]
I think the goal is to get the OpenSBI stuff in the next release so it can be tested a bit more.

**Comment by NathanTP** [comment]
There are too many open questions with OpenSBI right now, I also don't have enough time to get it working on the release timeline. Besides, I'd feel better if we had a chance to dogfood it a bit in case something unexpected stops working.

**Pull request merged**